clients/apps/web/src/pages/api/github/[org]/[repo]/issues/[number]/pledge.svg.tsx

```diff
@@ -93,6 +95,19 @@ export default async function handler(
     res.setHeader('Content-Type', 'image/svg+xml')
     res.setHeader('Cache-Control', 'no-cache')
     res.end(svg)
+
+    if (posthog) {
+      posthog.capture({
+        distinctId: randomUUID(),
```

Comment:
Here, we don't have a reasonable ID to use. We also don't have the org, repo or issue id